MapboxMobileEvents/MMEEventLogger.m

```diff
@@ -18,4 +26,44 @@ + (void)setEnabled:(BOOL)enabled {
     _enabled = enabled;
 }
 
+- (void)writeEventToLocalDebugLog:(MMEEvent *)event {
```

Comment:
I think we can go ahead and combine this new method and the old `logEvent`. I like the name `logEvent` because it is easier to say. We can just assume that if something gets logged it is written to the console and to disk.

Reply:
I definitely like this approach better. Initially I had similar thoughts with combining the two methods. My only concern was potentially having too many logs written locally and that a majority of the debug logs will have the same ``MMEEventKeyLocalDebug`` key.

It seems like we'll still need to have other keys for at least ``flush`` and ``push`` and that would likely result in any ``MMEEventKeyLocalDebug`` key/value pairs being unused; unless that information is interesting in some way...

Reply:
Let's go ahead and default to writing "too much" for now as a first pass. We can eventually introduce the notion of log levels to filter what actually gets printed and written.